pypln/web/settings.py

```diff
@@ -18,98 +18,96 @@
 }
 
-# Hosts/domain names that are valid for this site; required if DEBUG is False
-# See https://docs.djangoproject.com/en/1.5/ref/settings/#allowed-hosts
-ALLOWED_HOSTS = []
+MONGODB_CONFIG = config('MONGODB_CONFIG', default='mongodb://localhost:27017/pypln', cast=parse_url)
+
+
+MONGODB_CONFIG.update({
+    'analysis_collection': config('ANALYSIS_COLLECTION', default='analysis'),
+    'gridfs_collection': config('GRIDFS_COLLECTION', default='files'),
+    'monitoring_collection': config('MONITORING_COLLECTION', default='monitoring'),
+})
+
+ALLOWED_HOSTS = config('ALLOWED_HOSTS', cast=Csv())
 
-# Local time zone for this installation. Choices can be found here:
-# http://en.wikipedia.org/wiki/List_of_tz_zones_by_name
-# although not all choices may be available on all operating systems.
-# In a Windows environment this must be set to your system time zone.
 TIME_ZONE = 'America/Chicago'
 
-# Language code for this installation. All choices can be found here:
-# http://www.i18nguy.com/unicode/language-identifiers.html
 LANGUAGE_CODE = 'en-us'
 
 SITE_ID = 1
 
-# If you set this to False, Django will make some optimizations so as not
-# to load the internationalization machinery.
 USE_I18N = True
 
-# If you set this to False, Django will not format dates, numbers and
-# calendars according to the current locale.
 USE_L10N = True
 
-# If you set this to False, Django will not use timezone-aware datetimes.
 USE_TZ = True
 
-# Absolute filesystem path to the directory that will hold user-uploaded files.
-# Example: "/var/www/example.com/media/"
 MEDIA_ROOT = os.path.join(PROJECT_ROOT, 'media/')
 
-# URL that handles the media served from MEDIA_ROOT. Make sure to use a
-# trailing slash.
-# Examples: "http://example.com/media/", "http://media.example.com/"
 MEDIA_URL = '/media/'
 
-# Absolute path to the directory static files should be collected to.
-# Don't put anything in this directory yourself; store your static files
-# in apps' "static/" subdirectories and in STATICFILES_DIRS.
-# Example: "/var/www/example.com/static/"
 STATIC_ROOT = ''
 
-# URL prefix for static files.
-# Example: "http://example.com/static/", "http://static.example.com/"
 STATIC_URL = '/static/'
 
-# Additional locations of static files
 STATICFILES_DIRS = (
-    # Put strings here, like "/home/html/static" or "C:/www/django/static".
-    # Always use forward slashes, even on Windows.
-    # Don't forget to use absolute paths, not relative paths.
 )
 
-# List of finder classes that know how to find static files in
-# various locations.
 STATICFILES_FINDERS = (
     'django.contrib.staticfiles.finders.FileSystemFinder',
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
-#    'django.contrib.staticfiles.finders.DefaultStorageFinder',
 )
 
-# Make this unique, and don't share it with anybody.
-SECRET_KEY = 'louzsg2)(k8smz_jojk$y99nc-p11kia1unl9+yvvdvpel!z1b'
+SECRET_KEY = config('SECRET_KEY', default='louzsg2)(k8smz_jojk$y99nc-p11kia1unl9+yvvdvpel!z1b')
```

Comment:
Id change this to remove the default. The way things were, it was not possible to run with this (insecure) SECRET_KEY in a production environment because settings.production would complain about a non-existing secret_key file. With this setup, a user that misconfigures the application might end up using this SECRET_KEY, which is not a good idea at all.